### PR TITLE
LiveStreamSingle cinema layout

### PR DESCRIPTION
**Pull request by rkdavidson**
# About
- Refactors the `<Live>` layout to be "cinema style" by default — large, full bleed video and chat area with details below.
- Scaffolds out mobile responsiveness.
- Fixes shaka player video issue where there was too much height on iOS and Safari.
- Cleans up some dark mode / theming to prep for proper dark mode when viewing a live stream.

# Caveats / Tradeoffs
Some attention was given to fleshing out dark mode styles, but it's not complete. The `Header` looks particularly bad in dark mode.
I'm not 100% happy with the css fix for a shaka-player height but, it is a simple fix with some slightly confusing math/box model things to remember.

**Fullscreen** and some UX nuances for mobile aren't great, either.
If we autoplay the video, on mobile, it'll go into fullscreen right away. This feels jarring.
Disabling autoplay doesn't feel great for a livestream context, either. It adds unnecessary friction.
The temporary solution in this PR is to use browser `<video>` API for `playInline`, to prevent this auto-full screen behavior on load. **The tradeoff is** that it appears it disables the native player controls, so you cannot fullscreen at all.

Landscape mode also needs some love.

# Testing
Go to http://local.christfellowship.church:3000/live/test-event-24-hour-live-stream in both Desktop and Mobile.
I recommend opening `ThemeProvider` and changing the default mode to `dark`.
This change will be made automatically later.

**You probably should follow the API testing instructions from PR #59, too**.
This is because the test live streams are hidden by authenticated user security groups.

# Screenshots/Gifs
|Caption|🖼|
|---|---|
|Desktop|desktop|
|iPhone X|iphone|
|Resizing|resizing-new|
|Loading State|loading state|
|Grid (Desktop)|CleanShot 2021-03-10 at 12 12 04@2x|
|Grid (Mobile)|CleanShot 2021-03-10 at 12 12 17@2x|

### CODE REVIEW

File: components/Header/Header.styles.js [hunk 1]
```diff
@@ -4,10 +4,12 @@ import { themeGet } from '@styled-system/theme-get';
 import { system } from 'ui-kit';
 
 const Header = styled.header`
-  background-color: ${themeGet('colors.white')};
+  background-color: ${themeGet('colors.paper')};
+  color: ${themeGet('colors.fg')};
   box-shadow: ${themeGet('shadows.base')};
   align-items: center;
   justify-content: center;
+
```
[the remark this answers is not in the document]
Comment (rkdavidson) [reply components/Header/Header.styles.js:12]: Oh not particularly, I can remove it. Probably was trying something and undid it, but left the newline.

**Comment by drewbarontini** [comment]
@rkdavidson Is there a way to visit this on the Netlify deploy URL? I see the test event when signed in, but just get a 404 when trying to visit it.

**Comment by rkdavidson** [comment]
Got some feedback from Caleb and fixed newly broken Pre/Post states.


|Mobile Layout|Pre-Live|Post-Live|
|---|---|---|
||||

**Comment by drewbarontini** [comment]
@rkdavidson Ah, that makes sense! I'm not sure if there is a better workaround than that 🤷‍♂️

**Review by jordanwade — DISMISSED** [review]

**Review by jordanwade — APPROVED** [review]

**Pull request merged**